protostar.py

```diff
@@ -33,6 +32,9 @@ def cairo_file(file_path: str) -> Path:
 
 cmd_install_parser = root_subparsers.add_parser("install")
 cmd_install_parser.add_argument("package", type=str)
+cmd_install_parser.add_argument(
+    "--name", type=str, help="Custom package's directory name"
```

Comment:
is that argument required?

Reply:
It's optional. All named arguments are optional by default.